src/reaction/pade_approximant.cc

```diff
@@ -50,6 +50,9 @@ PadeApproximant::PadeApproximant(Input::Record in_rec)
     //DBGMSG("PadeApproximant constructor.\n");
     nominator_degree_ = in_rec.val<int>("nominator_degree");
     denominator_degree_ = in_rec.val<int>("denominator_degree");
+    if (nominator_degree_+1 != denominator_degree_ &&
```

Comment:
According to Flaherty the case denominator == nominator should be also stable and even more it should have maximal order of convergence. Thats why there was default value 2 for the nominator.  (2,2) Pade aproximant should be mid-point rule that is A-stable.